source/features/dim-bots.css

```diff
@@ -3,16 +3,16 @@ READ BEFORE EDITING. Some selectors match the same items ("focused" selector vs
 Commits that have a long commit message will not be dimmed after the user uncollapses the `<details>` element.
 */
 
-.rgh-dim-bot .commit-title, /* Pre "Repository refresh" layout */
-.rgh-dim-bot .commit-meta, /* Pre "Repository refresh" layout */
-.rgh-dim-bot .mb-1,
-.rgh-dim-bot .mb-1 ~ .d-flex,
-.rgh-dim-bot > .d-md-block,
-.rgh-dim-bot .commit-links-group, /* Hash and Copy hash buttons, pre "Repository refresh" layout */
-.rgh-dim-bot .commit-links-group + .btn, /* Browse repository button, pre "Repository refresh" layout */
-.rgh-dim-bot .Box-row--drag-hide, /* PR row */
-.rgh-dim-bot .labels, /* PR labels */
-.rgh-dim-bot .labels + .text-small /* PR meta */ {
+.rgh-dim-bot:not(.rgh-tagged) .commit-title, /* Pre "Repository refresh" layout */
```

Comment:
Lets drop the `/* Pre "Repository refresh" layout */` lines so that #3951 doesn't have to touch this file anymore, avoiding further conflicts

Reply:
@yakov116 you might want to _restore_ this file in your PR before I merge this, so no merge conflicts are ever shown

Reply:
I found that rebasing works best. I will debase that PR before I make it ready to review.